src/main/java/user_interface/login_leaderboard/Panel.java

```diff
@@ -1,8 +1,28 @@
 package user_interface.login_leaderboard;
 
 import javax.swing.*;
+import java.awt.*;
 
 public class Panel extends JPanel {
     public final int PANEL_WIDTH = 768;
     public final int PANEL_HEIGHT = 576;
+
+    public final Color TEXT_COLOR = new Color(255, 140, 0);
+
```

Comment:
There's a lot of unnecessary empty lines, it would be great if you can clean up a little